pkgs/development/python-modules/robotsuite/default.nix

```diff
@@ -10,7 +15,13 @@ buildPythonPackage rec {
   };
 
   buildInputs = [ unittest2 ];
-  propagatedBuildInputs = [ robotframework lxml ];
+  propagatedBuildInputs = [ robotframework lxml ]
+    ++ stdenv.lib.optional isPy3k robotframework-python3;
```

Comment:
From https://pypi.org/project/robotframework-python3/:

> Compatible with Python 2.7

So I think we should use `robotframework-python3` in both cases. This way we can get rid of `postPatch`.

Reply:
The package actually requires this (now outdated) fork for python3 because the mainline version was python2 only at the time. I'm not sure if we can use the latest robotframework for both python 2/3 builds.
I must have left the `postPatch` here after testing this. It does build but I don't know how  to properly test it.

Reply:
Ah, I see. Then we should simply patch `robotsuite`'s `setup.py`:
```
postPatch = ''
  substituteInPlace setup.py \
    --replace robotframework-python3 robotframework
'';

Reply:
Ok, done.